### DISCUSSION TITLE
Self-host on Raspberry Pi 4 using Docker

### DISCUSSION THREAD

**Discussion by redmouse112** [wger-project/wger#957] [Q&A] [answered]
Hi,

I would like to know if it is possible to self-host wger on a Raspberry Pi 4 using Docker.

On DockerHub there are arm64 containers but whenever I try to use them I get the following message:

`No matching manifest for linux/arm/v7 in the manifest list entries`

I'm sorry if it is a basic question. I'm still new to this. It seems it should be possible from the following tweet from the team:

https://nitter.net/wger_project/status/1442590063922139137

Thank you in advance and congrats for the amazing project.

**Comment by rolandgeider** [comment, marked as answer]
Hi! Glad you like it!

While I don't have one myself, I know from people from the community that it does indeed work on a raspberry. I'm just guessing here, but you might need to use a 64 OS image. On the other hand I could look and check what needs to be done to make images for arm/v7, it might just be a config flag

**Reply by redmouse112** [reply]
Thank you very much for the prompt reply!

Yes you are right, I think I need to use a 64 OS image. Hmm..I guess I'll wait a couple of days to decide what to do. At moment I'm running some other (important) services and it would take me some time to move everything to a new OS. 

Nevertheless, thanks!
